src/components/UI/pages/MenuPage.tsx

```diff
@@ -0,0 +1,20 @@
+import './MenuPage.css';
+
+import { Box, Button } from '@mui/material';
+
+import MenuItem from '../organisms/MenuItem/MenuItem';
+
+function MenuPage() {
+  return (
+    <Box className="menu-page">
+      <Button className="menu-page__order-button" variant="contained">
+        ORDER
+      </Button>
+      <MenuItem />
```

Comment:
For showing up it's okay but overall you know you could just use:

`[...Array(n)].map((elementInArray, index) => ( 
    <div className="" key={i}>RANDOM</div> 
    ) 
)`

Reply:
I wanted to have only one ORDER button at the top - not for every meal.

Reply:
This comment is for `MenuItem` not `Button`  😄